controllers/drplacementcontrol_controller.go

```diff
@@ -606,22 +606,29 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, errorswrapper.Wrap(err, "failed to get DRPC object")
 	}
 
+	drPolicy, err := r.getDRPolicy(ctx, drpc, logger)
```

Comment:
If DRPolicy is being deleted then we cannot process DRPC deletion, as this call would error out. Handling DRPC deletion post DRPolicy deletion is being worked on, and should address this as well, in case we do not address it here @ELENAGER

Reply:
I have reverted the DRPolicy validation code back.